lib/auth/auth_with_roles.go

```diff
@@ -1378,6 +1379,10 @@ func (a *ServerWithRoles) UpsertOIDCConnector(ctx context.Context, connector ser
 	if err := a.authConnectorAction(defaults.Namespace, services.KindOIDC, services.VerbUpdate); err != nil {
 		return trace.Wrap(err)
 	}
+	if modules.GetModules().Features().OIDC == false {
```

Comment:
Shouldn't this be checked first thing, i.e. even before all RBAC checks above, or is that intended? Same for checks below.

Reply:
I think we should give standard RBAC to kick in before build specific checks.